demos/react-jsnation/src/components/DataTable.tsx

```diff
@@ -12,110 +12,115 @@ LOGGER.level = LogLevel.Debug;
 export default function (props: { locale: string }): JSX.Element {
   const locale = useProp(props.locale, "props.locale");
 
-  return useReactiveSetup((component) => {
-    component.attach(DevTools);
+  return useReactiveSetup(
```

Comment:
All formatting changes below this line are due to me commenting out the `component` argument in the callback function. When we revert that change after fixing the devtool integration, prettier, will reformat it again.